modules/component/src/let/let.directive.ts

```diff
@@ -233,9 +235,5 @@ export class LetDirective<PO> implements OnInit, OnDestroy {
       this.isSuspenseViewCreated = true;
       this.viewContainerRef.createEmbeddedView(this.suspenseTemplateRef);
     }
-
-    if (this.isMainViewCreated || this.isSuspenseViewCreated) {
-      this.renderScheduler.schedule();
-    }
```

Comment:
There is no need to schedule render for suspense events at all, because suspense events are synchronous.